src/misc/manifest.webmanifest

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "web.dev",
+  "short_name": "web.dev",
+  "start_url": "/",
+  "display": "standalone",
```

Comment:
btw I realized recently that the display: standalone version of the existing pwa makes it hard to share links. We have a share button but it is really just a tweet button. We might want to think about adding additional options for sharing in the future.